### ISSUE TITLE
EditorConfig support

### ISSUE THREAD

**Issue by dgelessus** [omz/Pythonista-Issues#91]
http://editorconfig.org/

This would be useful for working on multiple projects with e. g. different indentation style without having to manually switch it in the settings.

**Comment by controversial** [comment]
Atom just detects things like indentation automatically and adjusts it every time you open a file or switch tabs.

**Comment by dgelessus** [comment]
Yes, that would be nice too. EditorConfig is better for open-source projects though, because it also applies to new files, so you have a consistent indentation style, and not whatever the initial author of the file liked most.

**Comment by controversial** [comment]
Ok. Yeah, I have worked with codebases that mix indentations horribly
